### PR TITLE
Add SDK Example

### PR DESCRIPTION
**Pull request by PabloSzx**
Add a much-needed example, also to be used for the new documentation.

image

image

### CODE REVIEW

File: example/pull-env.mjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { promises } from 'fs'
+import { fetch } from 'undici'
+
+await fetch('https://vault.dotenv.org/pull.txt', {
+  method: 'POST',
+  body: 'DOTENV_VAULT=vlt_4011aac495b93f704eea755ec8deba6c5eac6f1a6af55ba645a32b3015dce13c&DOTENV_ME=it_a106570406176c5a9233bd477704bd3c863b660e00cbd4f179023a11714cec70',
```
Comment (saihaj) [inline example/pull-env.mjs:6]: this is safe?

Comment (saihaj) [reply example/pull-env.mjs:6]: not everyone will be using `dotenv`. should we just add `.env.example` file

Comment (PabloSzx) [reply example/pull-env.mjs:6]: yep, I created a completely separate project and environment with this info

Comment (PabloSzx) [reply example/pull-env.mjs:6]: basically this is to not do `curl` and assume that's available everywhere, I would like SDK to work without any issue on windows for example

Comment (PabloSzx) [reply example/pull-env.mjs:6]: I also didn't want to install dotenv-vault since it's adding another dependency

Comment (saihaj) [reply example/pull-env.mjs:6]: > yep, I created a completely separate project and environment with this info

can we make that dotenv project public for others to import?

**Review by saihaj — APPROVED** [review]

File: example/pull-env.mjs (hunk 1 above)
Comment (PabloSzx) [reply example/pull-env.mjs:6]: 🤔 I think this approach is fine, just sharing the token to download from it

**Pull request merged**